Makefile

```diff
@@ -0,0 +1,16 @@
+##
+##usage :
+##-------
+
+build: ## build
```

Comment:
Not sure if this comment is needed eiter

Reply:
see https://github.com/EvanPurkhiser/tune-manager/pull/13#discussion_r520578195